AppCenterDataStorage/AppCenterDataStorage/Internal/Client/MSTokenExchange.h

```diff
@@ -19,10 +19,12 @@ typedef void (^MSGetTokenAsyncCompletionHandler)(MSTokensResponse *tokenResponse
  * @param completion callback that gets the token.
  *
  */
-+ (void)tokenAsync:(MSStorageIngestion *)httpIngestion
-           partitions:(NSArray *)partitions
++ (void)performDbTokenAsyncOperationWithHttpClient:(MSStorageIngestion *)httpIngestion
+           partition:(NSString *)partition
```

Comment:
there is a tool that checks the formatting:
to install: brew install clang-format
then run:  sh ./clang-format-changed-files.sh  from the root of iOS repository